source/adapters/hip/command_buffer.cpp

```diff
@@ -162,3 +163,41 @@ UR_APIEXPORT ur_result_t UR_APICALL urCommandBufferEnqueueExp(
                   "implemented for HIP adapter.");
   return UR_RESULT_ERROR_UNSUPPORTED_FEATURE;
 }
+
+UR_APIEXPORT ur_result_t UR_APICALL
+urCommandBufferRetainCommandExp(ur_exp_command_buffer_command_handle_t) {
+  detail::ur::die("Experimental Command-buffer feature is not "
```

Comment:
Hi Ewan just confirming as per https://github.com/oneapi-src/unified-runtime/pull/1089 if you could remove the `die` calls please, thank you!